docs/DatabaseCatalogAPI.yaml

```diff
@@ -253,14 +253,9 @@ paths:
                   results:
                     type: array
                     items:
-                      oneOf:
```

Comment:
New openapi codegen generates classes that changed the signature of the response. The new signature uses the discriminator, which can be handy in a few cases but is less intuitive to consume.